azext_iot/operations/dps.py

```diff
@@ -6,17 +6,22 @@
 
 from knack.log import get_logger
 from knack.util import CLIError
-from azext_iot.common.shared import (SdkType,
-                                     AttestationType,
-                                     ReprovisionType,
-                                     AllocationType)
+from azext_iot.common.shared import (
```

Comment:
since you linted this file can you point out the specific regions where your changes are so we can focus on what to look at?

Reply:
line 546 to 549